plugins/python-build/share/python-build/micropython-1.9.3

```diff
@@ -1,2 +1,2 @@
 #require_gcc
-install_package micropython-1.9.3 "https://github.com/micropython/micropython/releases/download/v1.9.3/micropython-1.9.3.tar.gz#f94c0b4834edca625b86c0846c89e609ee8fbdbe98e7f719d5108d74b6b4945e" micropython
+install_package micropython-1.9.3 "https://micropython.org/resources/source/micropython-1.9.3.tar.xz#3aa37065f5ea8df372d36253d0ae6333c68572e577df8558d4b7b93e070c624d" micropython
```

Comment:
Please wrap this in a check for `has_tar_xz_support`. You can see an exaple in [3.7.2](https://github.com/pyenv/pyenv/blob/ddb335c1979554196ccd18e7b2c26cc747cd4a89/plugins/python-build/share/python-build/3.7.2#L4).